mailpoet/tests/integration/Newsletter/Scheduler/PostNotificationTest.php

```diff
@@ -289,17 +304,25 @@ public function testSchedulerWontRunIfUnsentNotificationHistoryExists() {
     ]);
 
     $notificationHistory = new NewsletterEntity();
-    $notificationHistory->setType(Newsletter::TYPE_NOTIFICATION_HISTORY);
-    $notificationHistory->setStatus(Newsletter::STATUS_SENDING);
+    $notificationHistory->setType(NewsletterEntity::TYPE_NOTIFICATION_HISTORY);
+    $notificationHistory->setStatus(NewsletterEntity::STATUS_SENDING);
     $notificationHistory->setParent($newsletter);
     $notificationHistory->setSubject($newsletter->getSubject());
     $this->newslettersRepository->persist($notificationHistory);
     $this->newslettersRepository->flush();
 
-    $sendingTask = SendingTask::create();
-    $sendingTask->newsletterId = $notificationHistory->getId();
-    $sendingTask->status = SendingQueue::STATUS_SCHEDULED;
-    $sendingTask->save();
+    $task = new ScheduledTaskEntity();
```

Comment:
You can use [ScheduledTask factory](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/tests/DataFactories/ScheduledTask.php) here.